code/modules/client/preferences/species_features/mutants.dm

```diff
@@ -2,7 +2,15 @@
 	savefile_key = "feature_mcolor"
 	savefile_identifier = PREFERENCE_CHARACTER
 	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
-	relevant_species_trait = MUTCOLORS
+	relevant_inherent_trait = TRAIT_MUTANT_COLORS
+
+/datum/preference/color/mutant_color/is_accessible(datum/preferences/preferences)
+	if (!..(preferences))
+		return FALSE
+
+	var/species_type = preferences.read_preference(/datum/preference/choiced/species)
+	var/datum/species/species = new species_type
+	return !(TRAIT_FIXED_MUTANT_COLORS in species.inherent_traits)
```

Comment:
Couldnt you do initial(species.inherent_traits) ? Then you can avoid instancing the species type

Reply:
You can't use initial on lists. I tried using the GLOB.species_list datums but it broke shit.

Reply:
Ah, I thought it was bitflags